solutions/usaco-380.mdx

```diff
@@ -131,5 +131,115 @@ public class CrossCountrySkiing {
 
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define pb push_back
+
+int r, c;
+vector<vector<int>> grid; // grid stores the elevation
+
+// this will be true if the corresponding location
+// of the grid has been visited atleast once
+vector<vector<bool>> visited;
+
+// right bottom left top
```

Comment:
Expand on this a bit more

Reply:
@Aadit-Ambadkar  Is it better now?

Reply:
147 is better, could you also expand on 150?

Reply:
@Aadit-Ambadkar  ok

Reply:
@Aadit-Ambadkar  I hope this will do.